rsocket-core/src/commonMain/kotlin/io/rsocket/kotlin/internal/flow/RequestChannelRequesterFlow.kt

```diff
@@ -35,8 +35,7 @@ internal class RequestChannelRequesterFlow(
     override fun create(context: CoroutineContext, capacity: Int): RequestChannelRequesterFlow =
         RequestChannelRequesterFlow(payloads, requester, state, context, capacity)
 
-    @OptIn(ExperimentalCoroutinesApi::class)
-    override suspend fun collectTo(scope: ProducerScope<Payload>): Unit = with(state) {
+    override suspend fun collectImpl(collectContext: CoroutineContext, collector: FlowCollector<Payload>): Unit = with(state) {
```

Comment:
Is this the new guidance for accessing the coroutineContext? There have been various appraoches in past few years.

https://kotlinlang.org/docs/reference/coroutines/basics.html